### PR TITLE
[GNB] Added Lvl 100 Balance Opener

### PR DESCRIPTION
**Pull request by EYEZOexe**
image

### CODE REVIEW

File: XIVSlothCombo/Combos/CustomComboPreset.cs
Comment (zbee) [inline XIVSlothCombo/Combos/CustomComboPreset.cs:1]: Changing option IDs like this will break users' setups.

**Review by zbee — COMMENTED** [review]
At least in terms of the user config, I don't think this is ready.

File: XIVSlothCombo/Combos/CustomComboPreset.cs
Comment (EYEZOexe) [reply XIVSlothCombo/Combos/CustomComboPreset.cs:1]: how can I add another feature then without changing the other IDs? do I just pick a random one? I thought the ID's were the ordering

Comment (EYEZOexe) [reply XIVSlothCombo/Combos/CustomComboPreset.cs:1]: Nvm fixed

**Comment by EYEZOexe** [comment]
image

File: XIVSlothCombo/Combos/CustomComboPreset.cs
Comment (zbee) [reply XIVSlothCombo/Combos/CustomComboPreset.cs:1]: The next highest available ID within the same thousands block.

No, you can order with the Custom Attribute:
```cs
[CustomComboInfo("Feature", "Description", GNB.JobID, 1)]
//                                                    ^ this is the order
```
You can refer to [DNC](https://github.com/Nik-Potokar/XIVSlothCombo/blob/5a14682a4bbecb52378902fc34efcaffbf9b7bee/XIVSlothCombo/Combos/CustomComboPreset.cs#L868).

Comment (EYEZOexe) [reply XIVSlothCombo/Combos/CustomComboPreset.cs:1]: gotcha ill use that for next time in that case! is it good?

Comment (zbee) [reply XIVSlothCombo/Combos/CustomComboPreset.cs:1]: If it appears at the top of the list as it is now, I think the PR is fine now from the perspective of `CustomComboPreset`.
I'm not a GNB, so I won't speak to the rest.

Comment (EYEZOexe) [reply XIVSlothCombo/Combos/CustomComboPreset.cs:1]: I used RPR as an example

image

and now the ordering and position of code in `CustomComboPreset`

**Pull request closed without merging**